CHANGELOG.md

```diff
@@ -10,6 +10,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added: Command line arguments for exporting games. These commands are intended for advanced uses only.
 - Fixed: During generation, actions that involves multiple progressive pickups are now properly considered.
 
+### Cave Story
+
+#### Logic Database
+
+##### Ruined Egg Corridor
+
+- Added: Health requirements to the Dragon Sisters.
```

Comment:
the name of the boss is just "the Sisters"

Reply:
Initially wanted to make it clearer that its the boss. But not like there's anything else in ruined egg thats also named `sisters` and makes sense within the health requirement context.

Reply:
I think it's much more clear if we call the boss by its name instead of something that isn't its name lol